realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/UserTests.kt

```diff
@@ -412,9 +390,20 @@ class UserTests {
         user.logOut()
     }
 
-    // FIXME
-    @Ignore("Not implemented yet")
+    @Test
     fun isLoggedIn() {
+        var anonUser = app.login(Credentials.anonymous())
+        assertTrue(anonUser.isLoggedIn)
+
+        anonUser.logOut()
+        assertFalse(anonUser.isLoggedIn)
+
+
+        val user: User = app.registerUserAndLogin(TestHelper.getRandomEmail(), "123456")
```

Comment:
Does it anything to test with multiple users?

Reply:
Have modified the test case so we test with multiple users too

Reply:
Sorry for not being precise. I was just wondering why you both tested with anonymous and ordinary user. Isn't it redundant. But I guess does not hurt. Testing that the state is correct for overlapping _session_ sounds like a good test case thought, so please keep it.